src/messages/List/List.module.css

```diff
@@ -0,0 +1,114 @@
+article .wrapper {
+	max-width: 295px;
```

Comment:
List is wider in the specs, but it is wider than any other message element, should be clarified if intended